### PR TITLE
docs(support): adding Support page

### PR DESCRIPTION
**Pull request by markcaron**
## What I did

1. Added [Get support page](https://deploy-preview-1642--red-hat-design-system.netlify.app/support/)
2. Added `margin-block-end` styles to `h2`, `h3`, and `h4` element in `backpage.css` to make the vertical rhythm nicer, since without it, the headings were flush to the tiles.

## Testing Instructions

1. Compare to [Figma mockup](https://www.figma.com/design/s2jasqNAZ2LljpbbmfEPog/Support?node-id=0-1&m=dev)
2. Double check other backpages to make sure the vertical rhythm looks nice, and that the heading margins didn't get messed up.

Closes: #1443

### CODE REVIEW

File: docs/support/index.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    <p>Discussions about web components in general can be found in <strong>#list-web-components</strong>.</p>
+  </div>
+  <div>
+    <h3>Join office hours <rh-tag color="red" variant="outline">Internal Red Hat users only</rh-tag></h3>
+    <p>The Red Hat Design System team holds office hours weekly. The meeting time and link are in our <strong>#red-hat-design-system</strong> Slack channel bookmarks.</p>
+  </div>
+</div>
+
+
+### Email us
```
Comment (marionnegp) [inline docs/support/index.md:84]: I think this "Email us" section can be part of the "Contact us" section's grid.

**Review by marionnegp — COMMENTED** [review]
- [x] "Get support" heading at the very top should use sentence case.
- [x] Tile spacing between heading and body text looks like it's gone. I only see it on this page in the DP though.

I don't see the addition of `margin-block-end` to headings causing issues on other pages. Having 32px between headings and paragraphs seems like too much space though, but maybe that's because I'm used to seeing a 16px there. @coreyvickery, do you have a strong preference either way?

I'm seeing an inconsistency in grid gap values. I had to fix spacing in my mockup too. It should be 48px (`--rh-space-3xl`) for two column layouts, like the "Contact us" section's. I think this should be tackled in a separate issue/PR though.

File: docs/support/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  </rh-tile>
+  <rh-tile>
+    <h4 slot="headline"><a href="https://github.com/RedHat-UX/red-hat-design-system/issues/new?assignees=&labels=bug,docs&projects=&template=3-bug-report.yml&title=%5Bbug%5D+%3Ctitle%3E">Report a bug on ux.redhat.com</a></h4>
+    Submit an issue to fix bugs, like 404s, on ux.redhat.com.
+  </rh-tile>
+  <rh-tile>
+    <h4 slot="headline"><a href="https://github.com/orgs/RedHat-UX/discussions/new?category=design-library-feedback">Submit design library feedback</a></h4>
+    Create a discussion to request new features or fixes for the Red Hat Design System’s Figma library.
+  </rh-tile>
+</div>
```
Comment (zeroedin) [inline docs/support/index.md:66]: I think a reorder might help place the Bug fix first as to me that might be the most used, then followed by the feedback links.

```Suggestion
<div id="docs-help-grid" class="grid sm-three-columns">
  <rh-tile>
    <h4 slot="headline"><a href="https://github.com/RedHat-UX/red-hat-design-system/issues/new?assignees=&labels=bug,docs&projects=&template=3-bug-report.yml&title=%5Bbug%5D+%3Ctitle%3E">Report a bug on ux.redhat.com</a></h4>
    Submit an issue to fix bugs, like 404s, on ux.redhat.com.
  </rh-tile>
  <rh-tile>
    <h4 slot="headline"><a href="https://github.com/orgs/RedHat-UX/discussions/new?category=documentation-feedback">Send documentation feedback</a></h4>
    If you have general feedback about content on ux.redhat.com, start a discussion.
  </rh-tile>
  <rh-tile>
    <h4 slot="headline"><a href="https://github.com/orgs/RedHat-UX/discussions/new?category=design-library-feedback">Submit design library feedback</a></h4>
    Create a discussion to request new features or fixes for the Red Hat Design System’s Figma library.
  </rh-tile>
</div>
```

**Review by zeroedin — APPROVED** [review]
Given the noted by @marionnegp and myself are fixed this is LGTM

**Comment by coreyvickery** [comment]
@marionnegp Always 16px in between headings and body text.

File: docs/support/index.md (hunk 1 above)
Comment (markcaron) [reply docs/support/index.md:84]: Done

Comment (markcaron) [reply docs/support/index.md:84]: > Tile spacing between heading and body text looks like it's gone.

@marionnegp not sure what you meant.

File: docs/support/index.md (hunk 2 above)
Comment (markcaron) [reply docs/support/index.md:66]: Reordered.

**Comment by markcaron** [comment]
@marionnegp @coreyvickery, I changed the `margin-block-end` on all headings to be `16px` (in the `flow` CSS layer as @zeroedin requested).

File: docs/support/index.md (hunk 1 above)
Comment (marionnegp) [reply docs/support/index.md]: It looks correct now. Might have been a glitch on my end?

File: docs/styles/styles.css [hunk 3]
```diff
@@ -171,6 +171,10 @@
     margin-block: var(--rh-space-md, 8px);
   }
 
+  :where(h1, h2, h3, h4, h5, h6) {
+    margin-block-end: var(--rh-space-lg, 16px);
```
Comment (marionnegp) [inline docs/styles/styles.css:175]: @markcaron, this is a small edit that might be better to tackle in a different PR about docs spacing. If we have a heading + a heading, the space between them would be 32px.

Comment (markcaron) [reply docs/styles/styles.css:175]: @marionnegp if it's ok, maybe we can push this out as it is, and then follow quickly after with #1610 adjusting heading + heading combinations. I don't think this will aversely affect docs.

**Review by marionnegp — APPROVED** [review]
